src/extractors/chroma.js

```diff
@@ -0,0 +1,18 @@
+import { createChromaFilterBank } from './../utilities';
+
+export default function(args) {
+  if (typeof args.ampSpectrum !== 'object' ||
+        typeof args.chromaFilterBank !== 'object') {
+    throw new TypeError();
+  }
+
+  var chromagram = args.chromaFilterBank.map((row, i) =>
+    args.ampSpectrum.reduce((acc, v, j) => acc + v * row[j], 0)
+  );
+  var maxVal = Math.max.apply(null, chromagram);
```

Comment:
since we're transpiling, I'm pretty sure we could just do `Math.max(...chromagram)` here